views/dashboard/footer/components/AddCancelForm.tsx

```diff
@@ -0,0 +1,109 @@
+import { Dispatch, SetStateAction, useEffect, useState } from 'react'
```

Comment:
I genuinely dont understand this component. It's named `AddCancelForm` which doesnt tell me anything about what it's doing. Then the contents are all about Edit.